macropy/ikernel.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
```

Comment:
why the shebang? Is it supposed to load as a script?

Reply:
BTW, there's something I noticed only now, which needs one more bit of polish - loading the IPython configuration (if possible to do with reasonable effort).

Currently this kernel doesn't load the configuration, which means things like autocall (if the user has configured it) don't take effect in `imacropy`.

I'll look at it now, and shortly let you know how it went. In other respects I think this should already be feature-complete, which is why I submitted the PR at this time.

(Unless you find something missing - that's what the discussion is for :) )